lesson05/assignment/css/style.scss

```diff
@@ -1,6 +1,249 @@
+$quatenary-color: #ebe8e2;
+
+$min-width: 200px;
+
+@mixin transform($value) {
+  -webkit-transform: ($value);
+  -moz-transform: ($value);
+  -ms-transform: ($value);
+  transform: ($value);
+}
+@mixin transition($value) {
+  -webkit-transition: ($value);
+  -moz-transition: ($value);
+  -ms-transition: ($value);
+  transition: ($value);
+}
+
+%testing {
+  border: 5px solid blue;
+}
+
+%testing1 {
+  border: 5px solid red;
+}
+
+/* Page layout */
+//define grid area
+main {
+  nav {
+    grid-area: nav;
+  }
+  .side__navigation {
+    grid-area: side;
+  }
+  .main__description {
+    grid-area: content;
+  }
+}
+
+//grid template layout
+main {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  grid-template-areas:
+    "nav nav"
+    "side content";
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  background-color: $quatenary-color;
+  padding: 1rem 4rem;
+  h1 {
+    color: $primary-color;
+    font-size: 3rem;
+  }
+  p {
+    color: $tertiary-color;
+    font-style: italic;
+  }
+}
+
+.main {
+  /*navigation animation that is adopted from https://codepen.io/dghez/pen/Kwoper*/
+  padding: 0;
+  width: 100%;
+
+  nav {
+    background: $primary-color;
+    text-align: center;
+  }
+  i {
+    transition: transform 1s;
+  }
+  .main__navigation-item {
+    color: $quatenary-color;
+    position: relative;
+    list-style: none;
+    font-size: 0;
+    text-transform: uppercase;
+    display: inline-block;
+    padding: 0;
+    font-weight: bold;
+    text-align: center;
+    li {
+      font-size: 0.8rem;
+      display: inline-block;
+      position: relative;
+      padding: 15px 20px;
+      cursor: pointer;
+      z-index: 5;
+      min-width: $min-width;
+      font-size: 1rem;
+    }
+  }
+  li {
+    margin: 0;
+  }
+  .dropdown {
+    overflow: hidden;
+    list-style: none;
+    position: absolute;
+    padding: 15px 20px;
+    width: 200%;
+    div {
+      @include transform(translate(0, -200%));
+      @include transition(all 0.5s 0.1s);
+      position: relative;
+    }
+    li {
+      text-align: left;
+      display: block;
+      padding: 15px 20px;
+      width: 100%;
+      background: $secondary-color !important;
+      font-weight: normal;
+      text-transform: capitalize;
+      color: $quatenary-color;
+      @include transition(background-color 0.02s);
+    }
+    li:hover {
+      background: $tertiary-color !important;
+    }
+  }
+  #marker {
+    height: 6px;
+    background: $secondary-color !important;
+    position: absolute;
+    bottom: 0;
+    width: $min-width;
+    z-index: 2;
+    @include transition(all 0.35s);
+  }
+
+  @for $i from 1 through 4 {
+    .main__navigation-item li:nth-child(#{$i}) {
+      &:hover ul div {
+        @include transform(translate(0, 0));
+      }
+      &:hover i {
+        transform: rotate(1080deg);
+      }
+      &:hover ~ #marker {
+        @include transform(translate(#{(-1 + $i) * $min-width}, 0));
+      }
+    }
+  }
+  /*side menu*/
+  .side__navigation {
```

Comment:
You'll want to utilize media queries so your grid is responsive. On mobile, your left menu seems to takeover.